tools/distrib/python/grpcio_tools/setup.py

```diff
@@ -76,6 +77,10 @@
   if mac_target and (pkg_resources.parse_version(mac_target) <
 		     pkg_resources.parse_version('10.9.0')):
     os.environ['MACOSX_DEPLOYMENT_TARGET'] = '10.9'
+    os.environ['_PYTHON_HOST_PLATFORM'] = re.sub(
+        r'macosx-[0-9]+\.[0-9]+-(.+)',
+        r'macosx-10.9-\1',
+        util.get_platform())
```

Comment:
Yes they should be, and yes I kind of turned my brain off on this one after checking the output of just `grpcio`. After #7416 is in we can spin off a distrib build and verify all the things.